src/cmd/tools/dtest/docker/harness/query_api_test.go

```diff
@@ -139,6 +139,11 @@ func verifyStatus(expectedStatus int) resources.ResponseVerifier {
 			return fmt.Errorf("expeceted %v status code, got %v", expectedStatus, status)
 		}
 
+		contentType := headers["Content-Type"][0]
```

Comment:
Please make this safer with `value, ok := map[key]`, and also check the length of slice before accessing it.

Reply:
+1

Reply:
https://github.com/m3db/m3/pull/2917/commits/f8581442f5cdb07c43cecbfb3c785f4411f7634f#diff-391a291f1f91098bc731847f8843207e5327fe87a52a4599a45ac0cf4c5732e7R142-R144